uavcan/driver/python_can.py

```diff
@@ -0,0 +1,157 @@
+#
+# Copyright (C) 2019 UAVCAN Development Team  <uavcan.org>
+#
+# This software is distributed under the terms of the MIT License.
+#
+# Author: Jos Vos <jos.vos@demcon.nl>
+#
+
+from __future__ import division, absolute_import, print_function, unicode_literals
+import os
+import errno
+import time
+import threading
+from logging import getLogger
+
+from .common import DriverError, TxQueueFullError, CANFrame, AbstractDriver
+from .timestamp_estimator import TimestampEstimator
+
+try:
+    import queue
+except ImportError:
+    # noinspection PyPep8Naming,PyUnresolvedReferences
+    import Queue as queue
+
+logger = getLogger(__name__)
+
+try:
+    import can
+except ImportError:
+    logger.error("no module 'can' available, please install python-can")
+
+
+class PythonCan(AbstractDriver):
+    TX_QUEUE_SIZE = 1000
+
+    def __init__(self, interface, **_extras):
+        super(PythonCan, self).__init__()
+
+        # get bus from default config
+        try:
+            self._bus = can.Bus()

```

Comment:
This would fail in a very obscure way if the module couldn't be imported above. I suggest checking whether Python CAN is available early in the constructor, and raising a well-formed exception if it's not explaining what happened.